src/playback/summary.jsx

```diff
@@ -172,7 +182,7 @@ class PlaybackSummary extends React.Component {
       .where({ encoding, platform, size })
       .first();
 
-    if (score === 'bad') {
+    if (score === 'bad' || score === 'fail') {
       result.speeds.sort((a, b) => a.speed - b.speed);
 
       const speed = result.speeds.find(s => s.loss > 1);
```

Comment:
I am not sure which direction you are sorting; I assume you are looking for the lowest speed.  Do this with one expression:

```
selectFrom(result.speeds)
    .sort("speed")
    .filter(s=>s.loss>1)
    .first()
```